### PR TITLE
[ADD] Weblate Widget

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by rousseldenis** [comment]
I haven't tested macro. If anyone can check this.

@sbidoul @pedrobaeza 

Following this : https://github.com/OCA/stock-logistics-warehouse/pull/950

**Review by Yajo — CHANGES_REQUESTED** [review]
Add a test please.

**Comment by rousseldenis** [comment]
> Add a test please.

Arf, didn't there was some for that !

Ok

**Comment by rousseldenis** [comment]
@Yajo Done and :green_apple:

File: README.md.jinja [hunk 1]
```diff
@@ -8,9 +8,13 @@
         {%- endif -%}
     {%- endfor -%}
 {%- endmacro -%}
+{%- macro odoo_version_tr(version) -%}
```
Comment (Yajo) [inline README.md.jinja:11]:
```suggestion
{%- macro odoo_version_slug(version) -%}
```

This macro name is more explicit IMHO.

File: README.md.jinja [hunk 2]
```diff
@@ -8,9 +8,13 @@
...
+    {{- version | replace(".","-") -}}
+{%- endmacro -%}
 [![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/{{ repo_id(repo_slug) }}/{{ odoo_version }}.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-{{ repo_slug }}-{{ repo_id(repo_slug) }})
 [![Build Status](https://travis-ci.com/OCA/{{ repo_slug }}.svg?branch={{ odoo_version }})](https://travis-ci.com/OCA/{{ repo_slug }})
 [![codecov](https://codecov.io/gh/OCA/{{ repo_slug }}/branch/{{ odoo_version }}/graph/badge.svg)](https://codecov.io/gh/OCA/{{ repo_slug }})
+[![Translation Status](https://translation.odoo-community.org/widgets/{{ repo_slug }}-{{ odoo_version_tr(odoo_version) }}/-/svg-badge.svg)](https://translation.odoo-community.org/engage/{{ repo_slug }}-{{ odoo_version_tr(odoo_version) }}/?utm_source=widget)
```
Comment (Yajo) [inline README.md.jinja:17]:
```suggestion
[![Translation Status](https://translation.odoo-community.org/widgets/{{ repo_slug }}-{{ odoo_version_slug(odoo_version) }}/-/svg-badge.svg)](https://translation.odoo-community.org/engage/{{ repo_slug }}-{{ odoo_version_slug(odoo_version) }}/?utm_source=widget)
```

**Review by Yajo — COMMENTED** [review]
I suggest a small enhancement before merging.

**Comment by rousseldenis** [comment]
@Yajo changes done. @sbidoul This could be added before 14.0 branches creation ?

**Review by Yajo — APPROVED** [review]

**Comment by Yajo** [comment]
@sbidoul wanna merge?

**Pull request merged**